src/contracts/atlas/Atlas.sol

```diff
@@ -137,12 +137,20 @@ contract Atlas is Test, Factory {
         key = key.holdUserLock(uCall.to);
         bytes memory userReturnData = _executeUserOperation(uCall, executionEnvironment, key.pack());
 
-        bytes memory returnData;
+        bytes memory protocolReturnData;
         if (CallBits.needsPreOpsReturnData(callConfig)) {
-            returnData = preOpsReturnData;
+            protocolReturnData = preOpsReturnData;
         }
         if (CallBits.needsUserReturnData(callConfig)) {
-            returnData = bytes.concat(returnData, userReturnData);
+            protocolReturnData = bytes.concat(protocolReturnData, userReturnData);
+        }
+
+        bytes memory searcherReturnData;
```

Comment:
I think the name ``searcherReturnData`` implies that this is data returned by the searcher, which I don't believe is accurate.

Reply:
I think `searcherForwardData` is the better one, will change

Reply:
agreed - that's perfect.

Reply:
@Code0x2 you may need to update this w/ the changes to main before we can merge.